jekyll/_cci2/linux-cuda-images-support-policy.adoc

```diff
@@ -0,0 +1,108 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.x
+  - Server v3.x
+---
+= CircleCI Linux CUDA images support policy
+:page-layout: classic-docs
+:page-liquid:
+:page-description: CircleCI Linux CUDA images release, update, and deprecation policy
+:icons: font
+:toc: macro
+:toc-title:
+
+[#overview]
+== Overview
+
+This document outlines the xref:#[CircleCI Linux CUDA image] release, update, and deprecation policy and applies to our Linux CUDA Images.
```

Comment:
@JalexChen Is there an existing page in the docs that we can link to on here?

Reply:
This should be it I think: https://circleci.com/docs/using-gpu/#linux-gpu

Reply:
i think the link @rosieyohannan provided would be it, but i also think that we should be adding a CUDA page to the developer hub. The list of all images in `using-gpu` redirects [here](https://circleci.com/developer/images?imageType=machine), but none of the images for CUDA are actually listed there, and i think that can make things extremely confusing

Reply:
We can add the link to the dev hub page when it's ready 👍